src/group_action_error.jl

```diff
@@ -0,0 +1,171 @@
+### Basic checks if the action defined by the user is really a group action.
+"""
+    GroupActionError
+`GroupActionError` is an indicator that the group action defined by
+`(G, action, basis)` might be incorrect.
+
+If you receive `GroupActionError` while running functions from `SymbolicWedderburn`
```

Comment:
Why would it be advisable if you have already received a `GroupActionError` ?

Reply:
I meant this as an explanatory docstring for `GroupActionError` (maybe somebody is looking at the source code without encountering it in the wild). What do you suggest.

The other thing I was thinking was to change `check_group_action` into
```julia
    try
        # all the checks
    catch err
        if err isa GroupActionError
            @error "Group action seems to be ill defined; to check the correctness of an action you may run `check_group_action(G, action, basis, full_check=true)`."
        end
        rethrow(err)
    end
```
Then we could remove (or shorten) this docstring.

What do you think?

Reply:
What's the advantage over `@error` an rethrowing ?

Reply:
> If you receive `GroupActionError` while running functions from `SymbolicWedderburn`, check the error message for a counter example showing why this is not a group action.

> If you encounter inconsistent results it is advisable to run
>     `check_group_action(G, action, basis, full_check=true)`
> to check the correctness of the implementation of the action.

Reply:
yeah, I guess there's none.